src/zjs_ble.c

```diff
@@ -1283,6 +1282,25 @@ static ZJS_DECL_FUNC(zjs_ble_descriptor)
     return jerry_acquire_value(argv[0]);
 }
 
+void zjs_ble_cleanup()
+{
+    ble_connection_t *conn = ble_handle->connections;
+    while (conn) {
+        ble_connection_t *tmp = conn;
+        bt_conn_unref(tmp->bt_conn);
+        conn = conn->next;
+        zjs_free(tmp);
+    }
+    zjs_ble_free_services(ble_handle->services);
+    jerry_release_value(ble_handle->ble_obj);
+    zjs_free(ble_handle);
+    ble_handle = NULL;
+}
+
+static const jerry_object_native_info_t ble_module_type_info = {
+   .free_cb = zjs_ble_cleanup
+};
+
 jerry_value_t zjs_ble_init()
```

Comment:
These inits can also be made static. If `ENABLE_INIT_FINI` is unavailable `JERRYX_NATIVE_MODULE()` will expand to the definition of two global functions: `<module_name>_register()` and `<module_name>_unregister()`, which will add resp. remove the module from Jerry's linked list of native modules which it knows about. So, in the absence of `ENABLE_INIT_FINI` those can still be used to register the modules with Jerry.

Reply:
I added it to zjs_common.json.